libs/utils/utils.js

```diff
@@ -656,6 +656,21 @@ export function decorateLinks(el) {
         window.adobeIMS?.signIn();
       });
     }
+    const copyEvent = '#_evt-copy';
+    if (a.href.includes(copyEvent)) {
+      const link = a.href.split(copyEvent)[0];
+      const isConButton = ['EM', 'STRONG'].includes(a.parentElement.nodeName) || a.classList.contains('con-button');
+      if (!isConButton) a.classList.add('static', 'copy-link');
```

Comment:
Another thought:
How about if we have those events, we just emit events, and the consumer side code can add their logic accordingly? 

The only remark I would have: Since you change classes within the scope of this event, this won't work or introduce CLS. Can't the style part of this be authored otherwise? 🤔

Reply:
I think there is some misunderstanding around this as I am simply adding a bold style to the link if it is a not a button. According to the design, the link was kind of a bold static link. Since, I am not changing/replacing any class, I am not sure how could this cause CLS? Could you please elaborate more or let me know if I have understood this correctly?

Reply:
Nono, it's connected to https://github.com/adobecom/milo/pull/2804#discussion_r1738503680 - thinking about a possible future path on how we can handle this more generically. Maybe the discussion shouldn't happen on this PR, but it surfaces the issue 😁